drivers/serial/uart_gecko.c

```diff
@@ -682,10 +713,18 @@ DT_INST_FOREACH_STATUS_OKAY(GECKO_UART_INIT)
 #define GECKO_USART_IRQ_HANDLER(idx)
 #endif
 
+#ifdef CONFIG_PM_DEVICE
+#define GECKO_USART_PM_INIT(idx)					       \
+	PM_DEVICE_DT_INST_DEFINE(idx, uart_gecko_pm_action);
+#else
+#define GECKO_USART_PM_INIT(idx)
+#endif
```

Comment:
`PM_DEVICE_DT_INST_DEFINE()` already expands to `<nothing>` if `!CONFIG_PM_DEVICE`.